keel-sql/src/main/resources/db/changelog/20201208-fix-environment-last-verified-cascade.yml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+- changeSet:
+    id: fix-environment-last-verified-cascade
+    author: fletch
+    changes:
+    - dropForeignKeyConstraint:
+        baseTableName: environment_last_verified
+        constraintName: fk_environment_last_verified_environment
+    - addForeignKeyConstraint:
+        baseTableName: environment_last_verified
+        baseColumnNames: environment_uid
+        constraintName: fk_environment_last_verified_environment
+        referencedTableName: environment
+        referencedColumnNames: uid
+        referencesUniqueColumn: true
+        onDelete: CASCADE
```

Comment:
@emjburns this is the important bit, without the cascade the foreign key prevents us deleting rows from `delivery_artifact` or `environment` that are referenced from this table. With the cascade in place the rows on this table will get deleted automatically when the parent row is deleted.